pages/1.11/storage/mount-disk-resources/index.md

```diff
@@ -58,6 +58,12 @@ In this example, a disk resource is added to a DC/OS agent post-install on a run
         sudo rm -f /var/lib/mesos/slave/meta/slaves/latest
         ```
 
+    1. Remove checkpointed agent resources with this command:
+
+      ```bash
+      sudo rm -f /var/lib/mesos/slave/meta/resources
```

Comment:
LGTM. Can you ask @chhsia0  to take a look as well?

Also, do we need to improve the log that Mesos agent itself spits out on recovery failures, to keep up with this documentation?

Reply:
See https://issues.apache.org/jira/browse/MESOS-9046